web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/Plugin/Field/FieldWidget/ViewsBasicDefaultWidget.php

```diff
@@ -156,6 +161,24 @@ public function formElement(
       '#suffix' => '</div>',
     ];
 
+    $form['group_user_selection']['markup_post_entity_types'] = [
+      '#type' => 'markup',
+      '#markup' => '</div>',
+    ];
+
+    // @todo add validation for only one term.
```

Comment:
Let's leave this for now as I want to keep charing forward. But if this becomes and issue we can change this to a text input and create our own autocomplete callback. The entity_autocomplete type is doing all of the work for us, but it sucks that it does not have enough options.

Reply:
Yah, from that discussion on d.o, it seems that work is progressing. We could try applying that patch if you'd like?

Reply:
It's good for now. We just want something stable enough to get client feedback. We'll just keep things moving for now.